### PR TITLE
Documentation improvement - misc (smoe:Docs29 misc pc 5)

### PR DESCRIPTION
**Pull request by smoe**
The usual small bits that somehow annoyed me.  I read something - this time not something in weblate but the PDF of the German "Getting Started" document - and I try to fix what I recognize to be fixed, be it in weblate or GitHub.

The individual commits describe themselves.

### CODE REVIEW

File: docs/src/getting-started/getting-linuxcnc.adoc [hunk 1]
```diff
@@ -239,18 +239,14 @@ during the boot up.
...
-
-In addition, for experienced users who are familiar with Debian system
-administration (finding install images, manipulating apt sources, changing
-kernel flavors, etc), new installs are supported on following platforms:
-("amd64" means "64-bit", and is not specific to AMD processors, it will
-run on any 64-bit x86 system)
+The easiest, preferred way to install LinuxCNC is to use the Live/Install Image as described above.
+That method is as simple and reliable as we can make it, and is suitable for novice users and experienced users alike.
+However, this will typically replace any existing operating system.
+If hou have files on the target PC that you want to keep, then use one of the methods described in this section.
```
Comment (andypugh) [inline docs/src/getting-started/getting-linuxcnc.adoc:245]: "you"?

Comment (smoe) [reply docs/src/getting-started/getting-linuxcnc.adoc:245]: Maybe that is why the build failed :)

File: debian/rules.in [hunk 2]
```diff
@@ -96,9 +95,8 @@ override_dh_auto_install-indep:
 	rm -f $(DESTDIR)/usr/share/doc/linuxcnc/*_nb.pdf
 
 override_dh_installdocs-indep:
-	# The G-code quick reference html docs are built as
-	# `docs/html/${LANG}/gcode.html`.  We install them
-	# into `/usr/share/doc/@MAIN_PACKAGE_NAME@/` in each
+	# The G-code quick reference html docs are built as `docs/html/${LANG}/gcode.html`.
+	# We install them # into `/usr/share/doc/@MAIN_PACKAGE_NAME@/` in each
```
Comment (hansu) [inline debian/rules.in:99]:
```suggestion
	# We install them into `/usr/share/doc/@MAIN_PACKAGE_NAME@/` in each
```

File: docs/src/getting-started/getting-linuxcnc.adoc [hunk 3]
```diff
@@ -31,22 +30,23 @@ The outline of the process looks like this:
...
+This section describes some methods for downloading the Live/Install image.
 
 === Normal Download
 
-For x86 PCs Download the Live/Install CD by clicking here:
-
-https://www.linuxcnc.org/iso/linuxcnc_2.9.2-amd64.hybrid.iso
+Software for LinuxCNC to download is presented on the project's
+https://linuxcnc.org/downloads/[Downloads page].
+Most users will aim for the disk image Intel/AMD PCs, the URL
```
Comment (hansu) [inline docs/src/getting-started/getting-linuxcnc.adoc:41]:
```suggestion
Most users will aim for the disk image for Intel/AMD PCs, the URL
```

File: docs/src/getting-started/getting-linuxcnc.adoc [hunk 4]
```diff
@@ -368,9 +366,8 @@ kernel.
 
 === Installing on Raspbian 12
 
-. Download a stock Raspbian image to an SD card and install using the
-  [Raspberry Pi Imager App](https://www.raspberrypi.com/software/)
-. Boot the Pi and open a terminal. Download and run the linuxcnc installer
-  script as described above.
+Don't. The latencies are too bad with the default kernel and the PREEMPT_RT (the RT is important) kernel of Debian does not boot on the Pi (as of 1/2024).
```
Comment (hansu) [inline docs/src/getting-started/getting-linuxcnc.adoc:369]:
```suggestion
Don't do that. The latencies are too bad with the default kernel and the PREEMPT_RT (the RT is important) kernel of Debian does not boot on the Pi (as of 1/2024).
```

**Pull request merged**